themes/theme-b2c-tailwind/src/molecules/quantity-selector.css

```diff
@@ -0,0 +1,41 @@
+[data-store-quantity-selector-input] {
+}
+
+[data-store-quantity-selector-button] {
+}
+
+[data-store-icon] {
+}
+
+[data-store-quantity-selector] {
+}
+
+.quantitySelector [data-store-icon] {
+  @apply m-0;
+}
+
+.quantitySelector [data-store-quantity-selector-input]:focus-visible {
+  @apply outline-none;
+}
+
+.quantitySelector [data-store-quantity-selector-input] {
+  @apply text-center bg-transparent w-full h-full border-0 p-0;
+}
+
+.quantitySelector [data-store-quantity-selector-button] {
+  @apply bg-transparent w-full h-full p-0 flex items-center justify-center;
+}
+
+.quantitySelector {
+  @apply h-12 rounded bg-white flex flex-row justify-center items-center;
+  width: 8.125rem;
+  outline: 1px solid #c4c5ca;
+}
+
+.quantitySelector:hover {
+  outline: 0.125rem solid #2953b2;
+}
+
+.quantitySelector:focus-within {
+  box-shadow: 0 0 0 3px #fff, 0 0 0 5px #8db6fa;
+}
```

Comment:
just for curiosity sake: why `.classSelector` instead of `data-attributes`?

Reply:
That's only to keep one component styleless. Then by just adding the `.classSelector` to the other component, the default style is applied.

Reply:
Hmm, I thought you were trying to scope the css styles only for this component.

Reply:
That's exactly it, @hellofanny

Reply:
Nice! Although I like this approach, we aren't following this pattern on the other components. So I'm not sure if we should keep it 🤔